sql-select/inactive-customers.sql

```diff
@@ -0,0 +1,5 @@
+select *
+  from "customers"
+  where "isActive" = 'false';
+
+-- should false be in single quotes?
```

Comment:
Good question: booleans and numbers don't need quotes, but PostgreSQL will type-convert them from strings if they are included. Unlike JS, `'false'` will turn into `false` (instead of `true`).